### DISCUSSION TITLE
Connect PIA to a specific server

### DISCUSSION THREAD

**Discussion by ken8521** [qdm12/gluetun#1160] [Q&A] [answered]
I'm not sure if this is a feature request or I'm missing something.

I've successfully gotten gluetun working through PIA.  My issue is I connect, and I'll connect to say (for example) Sweden.  Then the next time the container restarts, I'll connect to say, Canada.

Is there a way to choose which server I connect to, or maybe just restrict it to US servers?

Thanks

**Comment by DcR-NL** [comment, marked as answer]
https://github.com/qdm12/gluetun/wiki/Private-internet-access

SERVER_REGIONS: Comma separated list of [regions](https://github.com/qdm12/gluetun/wiki/Private-internet-access#Servers)

**Reply by forsterkevin** [reply]
These links are broken!

**Comment by ken8521** [comment]
Thanks for the response

- SERVER_HOSTNAMES=us-florida.privacy.network  (or any other hostnam) seems to solve the problem

**Reply by qdm12** [reply]
Doesn't SERVER_REGIONS work as mentioned https://github.com/qdm12/gluetun/discussions/1160#discussioncomment-3727601 ?

**Reply by ken8521** [reply]
It didn't for me.  I didn't get the whole "comma separated" statement.  Does that mean it should be US,Florida?  It didn't make a lot of sense to as that isn't what was laid out in the chart.  I trued US Florida (which obviously failed due to the space) US\Florida and US/Florida.. none of which worked, but the hostname did.

**Reply by qdm12** [reply]
SERVER_REGIONS=US Florida

Should work, what error logged did you get? It should list all possible choices.

**Reply by ken8521** [reply]
> SERVER_REGIONS=US Florida
> 
> Should work, what error logged did you get? It should list all possible choices.

I'll have to look at it again later today.. If I recall it just connected to some random server.  I assumed it was due to the space in the US Florida
